instruments/toptica/topmode.py

```diff
@@ -137,11 +139,23 @@ def enable(self):
         @enable.setter
         def enable(self, newval):
             if not isinstance(newval, bool):
-                raise TypeError(
-                    "Laser emmission must be a boolean, got: {}".format(newval))
+                raise TypeError("Emission status must be a boolean, got: {}".
+                                format(type(newval)))
+            if not self.is_connected:
+                return
             self.parent.set(self.name + ":enable-emission", newval)
 
         @property
+        def is_connected(self):
+            """
+            Check whether a laser is even connected
+            """
+            if self.serial_number == 'unknown':
+                raise RuntimeError("Laser was not recognized by charm "
+                                   "controller. Is it plugged in?")
+            return True
+
+        @property
         def on_time(self):
             """
             Gets the 'on time' value for the laser
```

Comment:
Doc string confuses me, it makes me thing I'm getting back a value for how long the laser has been on for or will be on for.